### PR TITLE
Draft for minimalism

### PR DESCRIPTION
**Pull request by xuezzou**
Hi, Ben, I just finished implementing the basic layout! May you take a look?
Thanks!

### CODE REVIEW

File: src/components/arrow.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		position: relative;
+		top: -2px;
+		animation: ${props => (props.open ? spinReverse : spin)} 0.3s linear forwards;
+	}
+`;
+
+const Arrow = ({ open }) => (
+	<ArrowContainer open={open}>
+		<svg width="10" height="6" viewBox="0 0 10 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+			<path
```
Comment (cktang88) [inline src/components/arrow.js]: put this in an `arrow.svg` file?

File: src/components/bigLogo.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import InViewMonitor from 'react-inview-monitor';
+import MtSvgLines from 'react-mt-svg-lines';
+
+import '../../node_modules/animate.css/animate.min.css';
```
Comment (cktang88) [inline src/components/bigLogo.js:7]: there's probably a better way to import...

File: src/components/bigLogo.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+import LogoOnly from '../images/logoOnly.svg';
+import LogoVandyHacksText from '../images/textVandyHacks.svg';
+import Grid from '../images/gridLogo.svg';
+import LocationDateText from '../images/locationDateText.svg';
+
+const Container = styled.div`
+	width: 76%;
```
Comment (cktang88) [inline src/components/bigLogo.js:16]: 76 is a weird # lol

File: src/components/arrow.js (hunk 1 above)
Comment (xuezzou) [reply src/components/arrow.js]: Yes! I forgot we actually have an asset for that!

File: src/components/bigLogo.js (hunk 2 above)
Comment (xuezzou) [reply src/components/bigLogo.js:7]: Is trying to import from cdn a good choice?

File: src/components/bigLogo.js (hunk 3 above)
Comment (xuezzou) [reply src/components/bigLogo.js:16]: lol true these styling should be generalized later QAQ

**Comment by bencooper222** [comment]
I haven't looked at the code yet. However, my inclination is that we should have a v1, v2 and v3 branch to push each version to while we figure out how the hell we're going to blend them.

File: src/components/bigLogo.js (hunk 2 above)
Comment (bencooper222) [reply src/components/bigLogo.js:7]: I think the current version is the best way unless this works: https://www.npmjs.com/package/react-animated-css

File: src/components/schedule.js [hunk 4]
```diff
@@ -36,23 +36,23 @@ const BoxStyle = styled.div`
 	position: absolute;
 	top: 2em;
 	left: 6em;
-	width: 7em;
+	width: 13vh;
```
Comment (cktang88) [inline src/components/schedule.js:60]: just curious, why vh instead of em?

**Pull request closed without merging**